commit 181d9e8ef2bc42530ae227069f63014a287e5a4a
Author: Marcelo Tosatti <mtosatti@redhat.com>
Date:   Sat Jul 18 19:57:27 2009 -0300

    Fix event-trace compat with 2.6.30
    
    Fix undefined tracepoint symbols on 2.6.30 (this kernel version
    lacks event trace).
    
    Signed-off-by: Marcelo Tosatti <mtosatti@redhat.com>

Index: kvm-88/kvm/kernel/external-module-compat-comm.h
===================================================================
--- kvm-88.orig/kvm/kernel/external-module-compat-comm.h
+++ kvm-88/kvm/kernel/external-module-compat-comm.h
@@ -845,7 +845,7 @@ static inline struct file *eventfd_fget(
 
 #include <linux/tracepoint.h>
 
-#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,30)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,31)
 
 struct tracepoint;
 
